client/js/libs/handlebars/parse.js

```diff
@@ -102,6 +102,13 @@ function analyseText(text) {
 	// NOTE: channel prefixes should be RPL_ISUPPORT.CHANTYPES
 	var channelPrefixes = ["#", "&"];
 	var punctuations = ["'", "\"", ".", ",", "!", "?", "¿", "<", ">", "(", ")", "{", "}"];
+	var commonProtocols = [
```

Comment:
`mailto` won't match - url detection is based on `://`
never have seen `mumble`, `ts3server` urls, are they using `://'